.github/workflows/root-ci.yml

```diff
@@ -77,6 +77,29 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  conditional_job_check_files:
```

Comment:
I do not understand why the CI jobs do not start: is there perhaps a mistake in the yaml?

Reply:
@dpiparo @ferdymercury If you click on checks from github you'll see that the one workflow has a warning flag next to it. If you click on the workflow you'll see the following error message
```
[Invalid workflow file: .github/workflows/root-ci.yml#L393](https://github.com/root-project/root/actions/runs/12096633306/workflow)
You have an error in your yaml syntax on line 393
```
You seem to have made an error at line 393 of the workflow. If you can't work out what it is I will investigate further.